debugclient/getopt.c

```diff
@@ -0,0 +1,1258 @@
+/* Getopt for GNU.
+   NOTE: getopt is now part of the C library, so if you don't know what
+   "Keep this file name-space clean" means, talk to drepper@gnu.org
+   before changing it!
```

Comment:
Would there be a non-GPL version of this? I am sure the BSDs have something similar. I would like to avoid contamination with LGPL and GPL here.